Sources/PayPalDataCollector/PayPalDataCollectorEnvironment.swift

```diff
@@ -0,0 +1,16 @@
+import PPRiskMagnes
+
+/// Enum of environments to use with PayPalDataCollector
+public enum PayPalDataCollectorEnvironment {
+    case sandbox
+    case production
+
+    internal var magnesEnvironment: MagnesSDK.Environment {
```

Comment:
Swift defaults to internal if nothing is provided, so we can remove internal here. 😄

Reply:
+1